datafusion/core/src/physical_planner.rs

```diff
@@ -1552,10 +1549,10 @@ pub fn create_window_expr_with_name(
     e: &Expr,
     name: impl Into<String>,
     logical_input_schema: &DFSchema,
+    physical_input_schema: &Schema,
```

Comment:
Thanks @mustafasrepo 
I'm still thinking if we need to have a physical schema? This is misleading a bit, we always need to keep in mind why the schemas are not the same, what is their difference, etc.

Basically we have a check earlier in the planner that checks actual schemas from batch and from the planner

Reply:
Thanks @comphead, It seems that we really don't need physical_input_schema. Removed it, thanks for the suggestion.